source/cmake_modules/CompilationFlags.cmake

```diff
@@ -78,3 +78,13 @@ if(USE_SANITIZER)
     add_definitions("-DNO_FORCE_MEMORY_RELEASE")
     set(NAVITIA_ALLOCATOR "")
 endif(USE_SANITIZER)
+
+exec_program(proj OUTPUT_VARIABLE PROJ_LIB_VERSION RETURN_VALUE _result)
```

Comment:
As we were mentionning, using `proj` binary introduces a new dependency that is not really necessary. 
If we can grep the main header file to extract the version number, that would save us a lot of pain.

Reply:
OK